scripts/in_container/bin/install_java.sh

```diff
@@ -34,7 +43,7 @@ if command -v java; then
     exit 1
 fi
 
-DOWNLOAD_URL='https://download.java.net/openjdk/jdk8u41/ri/openjdk-8u41-b04-linux-x64-14_jan_2020.tar.gz'
+DOWNLOAD_URL='https://download.oracle.com/java/23/latest/jdk-23_linux-x64_bin.tar.gz'
```

Comment:
@VladaZakharova Why do we need to change the JDK distribution?

@potiuk is Oracle JDK 23 license (NFTC) Apache-compatible?

Reply:
Oh hi, sorry, I missed the message:)
this is (was) the latest one, and on old one it was not working

Reply:
> They may not be distributed
> Apache projects may not distribute Category X licensed components, in source or binary form; in ASF source code or in convenience binaries. As with the previous question on platforms, you can rely on the component if its license terms do not affect the Apache product's licensing. For example, using a GPL'ed tool during the build is okay, but including GPL'ed source code is not.

https://www.apache.org/legal/resolved.html#category-x